src/components/Header.tsx

```diff
@@ -44,14 +44,13 @@ function Header(props: HeaderProps) {
 					name="twitter:image"
 					content="https://res.cloudinary.com/degyjrpjj/image/upload/v1688584020/website/share/ugaacimkpgt6rsnecvds.png"
 				/>
-
-				{/* Beam Analytics */}
-				<script
-					src="https://beamanalytics.b-cdn.net/beam.min.js"
-					data-token={process.env.BEAM_DATA_TOKEN}
-					async
-				/>
 			</Head>
+			{/* Beam Analytics */}
+			<script
+				src="https://beamanalytics.b-cdn.net/beam.min.js"
+				data-token={process.env.BEAM_DATA_TOKEN}
+				async
+			/>
```

Comment:
J'ai deplacer le script en dehors de Head, recommendation: [No Script Tags In Head Component](https://nextjs.org/docs/messages/no-script-tags-in-head-component)